packages/web-app-search/src/index.ts

```diff
@@ -0,0 +1,48 @@
+/* istanbul ignore file */
+
+import SearchBar from './portals/SearchBar.vue'
+import App from './App.vue'
+import store from './store'
+import List from './views/List.vue'
+import { providerStore } from './service'
+import { bus } from 'web-pkg/src/instance'
+import { SearchProvider } from './types'
+import { Component } from 'vue'
+
+bus.on('app.search.register.provider', (provider: SearchProvider) => {
+  providerStore.addProvider(provider)
+})
+
+export default {
+  appInfo: {
+    name: 'Search',
```

Comment:
I think this is not rendered anywhere, but please use a dummy `$gettext` around the name, so that we can pick the string up for transifex. Like we did it e.g. here https://github.com/owncloud/ocis/blob/cc8273f048e36503fd78b7e185321eb8f949bdf6/accounts/ui/app.js#L7